process/elasticproc/elasticProcessor.go

```diff
@@ -503,6 +507,11 @@ func (ei *elasticProcessor) SaveTransactions(obh *outport.OutportBlockWithHeader
 		return err
 	}
 
+	err = ei.indexTokensHandler.IndexCrossChainTokens(ei.elasticClient, preparedResults.ScResults, buffers)
```

Comment:
Shouldn't we better have a separate `sovereignElasticProcessor`, which calls `elasticProc.SaveTransactions` + this func?

Can stay as it is, though, since this comes from run type comps

Reply:
Added in TODO MX-16499 for refactoring